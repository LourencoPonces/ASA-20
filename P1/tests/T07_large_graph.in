15,22
11
8
9
16
4
3
3
13
18
6
0
4
18
8
8
2 9
2 15
9 15
6 9
6 15
3 14
3 12
10 14
1 10
1 12
1 11
12 14
4 12
4 11
4 10
4 14
11 14
7 11
7 12
8 13
5 8
5 13