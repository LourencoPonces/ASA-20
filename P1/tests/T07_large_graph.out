11
18
9
16
18
18
8
18
18
8
8
8
18
8
8